packages/data-store/src/typeorm/index.ts

```diff
@@ -0,0 +1,47 @@
+import {DataSource} from 'typeorm';
+import {SDKConfigsEntity} from './entities/sdk-configs.entity';
+import {logger} from '../logger';
+import {DataStoreConfigs} from '../types/types';
+import {NetworkEntity} from './entities/network.entity';
+import {WalletEntity} from './entities/wallet.entity';
+import {DocumentEntity} from './entities/document.entity';
+import {DocumentTypeEntity} from './entities/document-type.entity';
+
+let _dataSource: DataSource;
+
+export async function initializeTypeORM(options: DataStoreConfigs) {
+  let databasePath = 'wallet-data-store.sqlite';
+
+  if (process.env.NODE_ENV === 'test') {
```

Comment:
bit of a nitpick, but rather than expecting the datastore to know that tests are a thing at all, why not make DATASTORE_PATH an env var that defaults to `wallet-data-store.sqlite` but tests can set to `:memory:`?

Reply:
yeah, much better. I've pushed a commit for that